e2e/index.ts

```diff
@@ -39,4 +40,14 @@ describe("Test", async () => {
       "birthDate"
     );
   });
+
+  it("Should Login", async () => {
```

Comment:
```suggestion
  it("Should Login successfully given a valid input", async () => {
```
além disso, seria legal ter testes para quando dar errado, sabe? algo do tipo `should throw InputError when birthDate is undefined`

cc: @alanraso @matheusflds